tests/data/licenses.json

```diff
@@ -0,0 +1 @@
"note":"OSI Page","url":"https://opensource.org/licenses/VSL-1.0"}],"name":"The Vovida Software License, Version 1.0","other_names":[],"superseded_by":null,"keywords":["discouraged","non-reusable","osi-approved"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/VSL-1.0"}]},{"id":"W3C","identifiers":[{"identifier":"W3C","scheme":"SPDX"}],"links":[{"note":"OSI Page","url":"https://opensource.org/licenses/W3C"}],"name":"The W3C Software Notice and License","other_names":[],"superseded_by":null,"keywords":["discouraged","non-reusable","osi-approved"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/W3C"}]},{"id":"WXwindows","identifiers":[{"identifier":"WXwindows","scheme":"SPDX"}],"links":[{"note":"OSI Page","url":"https://opensource.org/licenses/WXwindows"}],"name":"The wxWindows Library Licence","other_names":[],"superseded_by":null,"keywords":["discouraged","non-reusable","osi-approved"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/WXwindows"}]},{"id":"Watcom-1.0","identifiers":[{"identifier":"Watcom-1.0","scheme":"SPDX"}],"links":[{"note":"OSI Page","url":"https://opensource.org/licenses/Watcom-1.0"}],"name":"The Sybase Open Source Licence","other_names":[],"superseded_by":null,"keywords":["discouraged","non-reusable","osi-approved"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/Watcom-1.0"}]},{"id":"Xnet","identifiers":[{"identifier":"Xnet","scheme":"SPDX"}],"links":[{"note":"OSI Page","url":"https://opensource.org/licenses/Xnet"}],"name":"The X.Net, Inc. License","other_names":[],"superseded_by":null,"keywords":["osi-approved","discouraged","redundant"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/Xnet"}]},{"id":"ZPL-2.0","identifiers":[{"identifier":"Zope-2.0","scheme":"DEP5"},{"identifier":"ZPL-2.0","scheme":"SPDX"}],"links":[{"note":"OSI Page","url":"https://opensource.org/licenses/ZPL-2.0"}],"name":"The Zope Public License, Version 2.0","other_names":[],"superseded_by":null,"keywords":["discouraged","non-reusable","osi-approved"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/ZPL-2.0"}]},{"id":"Zlib","identifiers":[{"identifier":"Zlib","scheme":"DEP5"},{"identifier":"Zlib","scheme":"SPDX"}],"links":[{"note":"OSI Page","url":"https://opensource.org/licenses/Zlib"}],"name":"The zlib/libpng License (Zlib)","other_names":[],"superseded_by":null,"keywords":["osi-approved"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/Zlib"}]},{"id":"jabberpl","identifiers":[],"links":[{"note":"OSI Page","url":"https://opensource.org/licenses/jabberpl"}],"name":"Jabber Open Source License","other_names":[],"superseded_by":null,"keywords":["discouraged","retired","osi-approved"],"text":[{"media_type":"text/html","title":"HTML","url":"https://opensource.org/licenses/jabberpl"}]}]
```

Comment:
At some point, I might mock this out into a handful of fake data, but for now, I think this is perfect.